doc/chips/9.rst

```diff
@@ -446,6 +449,9 @@ being installed as root (or at least becomes much more complicated if it is
 not). Nix also only works on linux/osx right now. Cygwin support is not ruled
 out, but not actively worked on.
 
+Another potential candidate that has been suggested is Conan_, but we have not
+had time to thoroughly investigate it yet.
```

Comment:
maybe 'not have had time' to just 'not yet investigated it'? It'd be silly if this line stayed here for months or years ..